Userland/Libraries/LibWeb/Layout/LayoutState.cpp

```diff
@@ -246,8 +246,12 @@ void LayoutState::UsedValues::set_node(NodeWithStyleAndBoxModelMetrics& node, Us
                 resolved_definite_size = adjust_for_box_sizing(size.calculated().resolve_length_percentage(node, containing_block_size_as_length).value_or(CSS::Length::make_auto()).to_px(node), size, width);
                 return true;
             }
-            resolved_definite_size = adjust_for_box_sizing(size.calculated().resolve_length(node)->to_px(node), size, width);
-            return true;
+
+            auto maybe_length = size.calculated().resolve_length(node);
+            if (maybe_length.has_value()) {
+                resolved_definite_size = adjust_for_box_sizing(maybe_length.value().to_px(node), size, width);
+                return true;
+            }
```

Comment:
This looks like a workaround. What's the problem here?
Do we have a `calc()` value that doesn't resolve to a length?
If so, how did it end up in one of the size properties (width/height/min-*)?

Also please add tests when doing bug fixes so we can understand better what they are meant to be fixing. :)

Reply:
Tried to reproduce this on latest commit, but i couldnt.. So it seems its been fixed by something else already 🤷